spec/frise/validator_spec.rb

```diff
@@ -214,6 +214,62 @@ def validators.short_string(_, str)
     expect(errors).to eq ['At key: {"c"=>"abc"} does not match any of the possible schemas']
   end
 
+  it 'should validate correctly $constant schemas' do
+    schema = { 'key' => { '$constant' => 42 } }
+
+    conf = { 'key' => 42 }
+    errors = validate(conf, schema)
+    expect(errors).to eq []
+
+    conf = { 'key' => 43 }
+    errors = validate(conf, schema)
+    expect(errors).to eq ['At key: invalid value 43. The only accepted value is 42']
+  end
+
+  it 'should make $constant schemas force values to be present' do
+    schema = { 'key' => { '$constant' => 42 }, '$allow_unknown_keys' => true }
+
+    conf = { 'key1' => 42 }
```

Comment:
Could we rename this to something like `other_key`? It was probably my own bad, but I spend like 20 seconds looking at the test confused because I was reading `key` and `key1` the same 😅 

```suggestion
    conf = { 'other_key' => 42 }
```

Reply:
Yes, of course!